test/end-to-end/server-external-fixtures-test.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+npm link .
+octokit-fixtures-server --fixtures $(pwd)/scenarios/api.github.com/get-organization/normalized-fixture.json &
+serverPid=$!
+
+sleep 3
+
+# should succeed since we're loading the get-organization fixture
+curl -H'Accept: application/vnd.github.v3+json' http://localhost:3000/orgs/octokit-fixture-org | grep -q '"login":"octokit-fixture-org"'
+returnCode1=$? # is 0 if grep could not find string above
+
+# should fail since we're not loading the repo (inverted due to grep -v)
+curl -H'Accept: application/vnd.github.v3+json' http://localhost:3000/repos/octokit-fixture-org/hello-world | grep -v -q '"full_name":"octokit-fixture-org/hello-world"'
+returnCode2=$? # is 1 if grep could not find string above
+
+kill $serverPid
+! (( $returnCode1 || $returnCode2 ))
```

Comment:
My bash skills are ... limited :) This looks great though, thank you!

Could you add a comment to line 18, I’m not 100% sure what it does.

And do you think that we could somehow run both end-to-end test with the `npm run test:e2e` script? We might add more in future?

Reply:
> Could you add a comment to line 18, I’m not 100% sure what it does.
Absolutely 👍 

> And do you think that we could somehow run both end-to-end test with the npm run test:e2e script? We might add more in future?

We could, though I think it's going to get confusing as we additional tests, because the ultimate exit code of the script needs to be an aggregate of all the tests within (e.g. it will get confusing when the last line of the script is "exit 0 if this succeeded and exit 1 if this failed and exit 2 if this other thing succeeded" etc).

I figure the "external fixtures" test probably won't have any more conditions, since it's literally just saying "load this using the external fixtures argument, and be sure nothing else is loaded." Any other new tests would probably just go in the regular `e2e` test, unless we wanted to drop the builtin, automagical loading of fixtures, and always require an argument for the fixtures.